### ISSUE TITLE
NoMethodError: undefined method `start_with?' for nil:NilClass

### ISSUE THREAD

**Issue by splattael** [asterite/rufo#64]
## Reproduction

```
  test "test "do
      post :x, y: {
                       :foo1 => :bar,

                       :foo2 => bar,
                     }

      multiline :call,
                       :foo => :bar,
                       :foo => bar
  end
```

Some notes:
* It works without empty line between `:foo1` and `:foo2`
* It works if `multline` is not there or just one line
* Without the `test` block it gives another error
    * `NoMethodError: undefined method '[]' for nil:NilClass` (see below)

## Error

```
NoMethodError: undefined method `start_with?' for nil:NilClass
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3288:in `block (2 levels) in do_align'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3285:in `any?'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3285:in `block in do_align'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `<<'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `each'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `each'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `do_align'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3264:in `do_align_hash_keys'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:209:in `format'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:112:in `format'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:68:in `format_file'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:58:in `block in format_args'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:57:in `each'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:57:in `format_args'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:19:in `run'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:6:in `run'
  exe/rufo:4:in `<top (required)>'
```

## Reproduction w/o `test` block

```
      post :x, y: {
                       :foo1 => :bar,

                       :foo2 => bar,
                     }

      multiline :call,
                       :foo => :bar,
                       :foo => bar
```

## Error w/o `test` block

```
NoMethodError: undefined method `[]' for nil:NilClass
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3286:in `block (2 levels) in do_align'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3285:in `any?'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3285:in `block in do_align'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `<<'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `each'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `each'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3279:in `do_align'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:3264:in `do_align_hash_keys'
  /opt/ps/home/devel/_private/rufo/lib/rufo/formatter.rb:209:in `format'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:112:in `format'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:68:in `format_file'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:58:in `block in format_args'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:57:in `each'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:57:in `format_args'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:19:in `run'
  /opt/ps/home/devel/_private/rufo/lib/rufo/command.rb:6:in `run'
  exe/rufo:4:in `<top (required)>'
```

@asterite I hope it's not overwhelming and thank you for `rufo` :heart: :heart: :heart: 

Refs #63

**Comment by asterite** [comment]
Fixed in 11be64d45324ae94718ccd6f38349947c9e52574

**Comment by splattael** [comment]
:+1: Awesome, thank you!

BTW, I am trying to introduce `rufo` to our projects :sunglasses:
